airflow/dags/gtfs_loader/calitp_files_process.py

```diff
@@ -0,0 +1,42 @@
+# ---
+# python_callable: main
+# provide_context: true
+# external_dependencies:
+#   - gtfs_downloader: download_data
+# ---
+
+from calitp import read_gcfs, get_bucket, save_to_gcfs
+import pandas as pd
+import gcsfs
+
+
+def main(execution_date, **kwargs):
+    # TODO: remove hard-coded project string
+    fs = gcsfs.GCSFileSystem(project="cal-itp-data-infra")
+
+    bucket = get_bucket()
+
+    f = read_gcfs(f"schedule/{execution_date}/status.csv")
+    status = pd.read_csv(f)
+
+    success = status[lambda d: d.status == "success"]
+
+    gtfs_files = []
+    for ii, row in success.iterrows():
+        agency_folder = f"{row.itp_id}_{row.url_number}"
+        gtfs_url = f"{bucket}/schedule/{execution_date}/{agency_folder}/*"
```

Comment:
how does this handle multiple files per agency?

Reply:
Ah, sorry--one thing I should have included in the notes is that the BigQuery can only handle a single wildcard in a bucket path. To avoid the need to constantly use `fs.glob` (which can do multiple wildcards), I changed the paths to be {itp_id}_{url_number}. 

I think it's only an issue because we have 2 calitp specific table keys in this case. Shouldn't be an issue for the GTFS-RT data, since it only has 1 key (itp_id), and 3 tables / blobs (e.g. vehicle_positions).

[Relevant bigquery docs](https://cloud.google.com/bigquery/docs/batch-loading-data#load-wildcards)

(edited this detail in to the top post just now, so it's easy to find)

Reply:
ahhh makes sense

Reply:
just for any joins etc need to do a combo of {key * ID number * url number}?

Reply:
for now this is just Metro too so not a big issues.